jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/inputs/ButtonTests.java

```diff
@@ -17,14 +17,4 @@ public void defaultButtonTest() {
         buttonFrame.button.click();
         buttonFrame.button.has().text("BUTTON");
     }
-
-    @Test(expectedExceptions = {RuntimeException.class},
-            expectedExceptionsMessageRegExp = ".*(Can't perform click. Element is disabled)")
-    public void disabledButtonTest() {
-        inputButtonDisabledPage.shouldBeOpened();
-
-        buttonFrame.button.is().disabled();
-        buttonFrame.button.click();
-    }
-
```

Comment:
Without this test no check for disabled button at all

Reply:
Runtime errors processed in core

Reply:
Can't see check for disabled button in ButtonTests.java

Reply:
fixed without runtime exception